src/main/java/com/minecolonies/entity/ai/minimal/EntityAICitizenWander.java

```diff
@@ -35,17 +35,17 @@ public boolean shouldExecute()
         {
             return false;
         }
-        Vec3 vec3 = RandomPositionGenerator.findRandomTarget(citizen, 10, 7);
-        if (vec3 == null)
+        Vec3d Vec3d = RandomPositionGenerator.findRandomTarget(citizen, 10, 7);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00117)